README.md

```diff
@@ -236,9 +239,13 @@ curl -X DELETE "http://localhost:9000/retry_test/1d05c20627844214a9ff7cbcf696317
 ### Causing a failure using x-retry-test-id header
 
 The following cURL request will attempt to list buckets and the testbench will emit
-a `503` error once based on the Retry Test created above. Subsequent list buckets
+a `503` error once in the JSON server based on the Retry Test created above. Subsequent list buckets
 operations will succeed.
 
+In this example, list buckets operations in the gRPC server will not be impacted since the Retry Test
+created above only applies to JSON. Specify `"transport": "GRPC"` for a Retry Test Resource to apply
+to gRPC operations.
+
```

Comment:
So, it looks like the errors/instructions are the same in gRPC vs. JSON. I'm wondering if this is really what we want. Since gRPC has a well definite set of error codes (even a broken stream will be turned into a specific gRPC error in the gRPC client-side code) I'm wondering if we should be accepting these codes in the retry test API rather than mapping from HTTP and/or doing stuff like forcing a broken stream. Thoughts?

Reply:
I'm under the impression that we wanted to use the same retry conformance test schema to run tests under both transports, and have that outlined in the design sketch. One of the biggest pros is that it relieves the refactoring burden on the clients. I think this remains a good starting point for conformance testing (i.e. retry for UNAVAILABLE; does not retry for UNAUTHENTICATED etc)

In the current design and implementation, there is a separate retry handler for the grpc server, so that will give us flexibility. We can further define what failure injections make more sense in the gRPC context. Scheduled some time with you to discuss offline :)